### PR TITLE
information update

### PR DESCRIPTION
**Pull request by joinimran**
As a request, the required information has been updated in the doc.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5261

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/bulk-enrollment-using-windows-provisioning-tool.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ On the desktop and mobile devices, you can use an enrollment certificate or enro
 > -   Bulk-join is not supported in Azure Active Directory Join.
 > -   Bulk enrollment does not work in Intune standalone environment.
 > -   Bulk enrollment works in System Center Configuration Manager (SCCM) + Intune hybrid environment where the ppkg is generated from the SCCM console.
-
- 
+> -   To change bulk enrollment settings, login to **AAD** then **Devices** and then click **Device Settings**. Change the number under **Maximum number of devices per user**.
```
Comment (illfated) [inline windows/client-management/mdm/bulk-enrollment-using-windows-provisioning-tool.md]:
```suggestion
> -   To change bulk enrollment settings, login to **AAD**, then **Devices**, and then click **Device Settings**. Change the number under **Maximum number of devices per user**.
```

**Comment by illfated** [comment]
Since I don't have any experience using this interface, I don't have any opinions worth mentioning regarding the phrasing.

**Comment by joinimran** [comment]
Ready to go live.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**